package.json

```diff
@@ -24,6 +24,7 @@
   },
   "homepage": "https://github.com/WilliamVelazquez/epets-api#readme",
   "dependencies": {
+    "cors": "^2.8.5",
```

Comment:
Remember to lock the dependency version to avoid future conflicts with other collaborator. (Just delete the ^ symbol).